packages/clap-host/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgname=clap-host
+pkgver=1.0.2
+pkgrel=1
+pkgdesc='CLAP example host'
+arch=(aarch64 x86_64)
+url='https://github.com/free-audio/clap-host'
+license=(MIT)
+depends=(gcc-libs qt6-base)
+makedepends=(catch2 cmake rtaudio rtmidi)
+groups=(pro-audio)
+_clap_ref='27b7af0'
+_clap_helpers_ref='4d4e297'
+source=("$pkgname-$pkgver.tar.gz::https://github.com/free-audio/$pkgname/archive/refs/tags/$pkgver.tar.gz"
+        "clap-$_clap_ref.tar.gz::https://github.com/free-audio/clap/archive/$_clap_ref.tar.gz"
```

Comment:
Can or should this not be compiled against the system installed `clap` package? Or is this tied to a specific version of CLAP?

Reply:
Compilation seems to work with clap 1.1.3. However clap-host's CMakeLists.txt adds the clap dependency using `add_subdirectory`, expecting the submodule instead of searching global CMake modules.

We could probably patch that but I'm not that experienced with CMake :/

Reply:
Ok, then I guess we stick to including clap in he source, even though upstream should then really package it in the source archive.